components/eam/docs/tech-guide/chemUCIlinozv3.md

```diff
@@ -0,0 +1,28 @@
+# chemUCI and Linoz v3
+
+## Overview
+
+Atmospheric interactive chemistry is handled by chemUCI (in the troposphere) and Linoz v3 (in the stratosphere). chemUCI consists of 28 advected tracers for the O3-CH4-HOx-NOx-NMVOCs chemistry. Compared to E3SMv2, the E3SMv3 linearized stratospheric chemistry scheme (Linoz v3) expends the interactive species to include O3, N2O, NOy, and CH4. The boundary between stratosphere and troposphere adopts the e90 tropopause algorithm.
+
+## Namelist parameters
+
+| Parameter                    | Description                                                              | Default value*         |
+| ---------------------------- | ------------------------------------------------------------------------ | ---------------------- |
+| `airpl_emis_file`            | Aviation emission                                                        |                        |
+| `chlorine_loading_file`      | Chlorine loading                                                         |                        |
+| `chlorine_loading_fixed_ymd` | Directory of P3 look-up tables                                           |                        |
+| `chlorine_loading_type`      | P3 look-up table Version                                                 |                        |
```

Comment:
The descriptions are for P3, not for these chlorine-related settings. The variable names should be self-explanatory because of the chlorine loading description above. It seems reasonable to leave these blanks empty.